tests/unit/config/interpolation_test.py

```diff
@@ -146,3 +146,9 @@ def test_interpolate_missing_with_default(defaults_interpolator):
 def test_interpolate_with_empty_and_default_value(defaults_interpolator):
     assert defaults_interpolator("ok ${BAR:-def}") == "ok def"
     assert defaults_interpolator("ok ${BAR-def}") == "ok "
+
+
+def test_interpolate_with_chained_fallback_variables(defaults_interpolator):
+    assert defaults_interpolator("ok ${BAR:-$FOO}") == "ok first"
+    assert defaults_interpolator("ok ${BAR:-$BAZ:-$FOO}") == "ok first"
+    assert defaults_interpolator("ok ${BAR-$FOO}") == "ok "
```

Comment:
Perhaps add another case to check coalesced interpolation:

`assert defaults_interpolator("ok ${BAR:-the $FOO def}") == "ok the first def"`

Reply:
This shomehow conflicts whith the previows review, not?

How about
```python
assert defaults_interpolator("ok ${BAR:-the ${FOO} def}") == "ok the first def"
```
Is it ok?

Reply:
Yes - depends on if the previous comment.